tests/job_concurrency1_test.py

```diff
@@ -9,9 +9,16 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 
 """
-from __future__ import absolute_import, division, print_function
 import pytest
 
+import sirepo.util
+
+def setup_module(module):
```

Comment:
setup_module means this should probably be a separate test file. The other cases will be affected by this.

More importantly, I don't think the new case is hitting the asynchrony problems. We want to test whether IO gets through when an Analysis is running. That's not what the case is test.

I'm also concerned about `fc` being used in multiple threads. I don't know enough about how flask client is constructed. It may contain state related to the request. I think for asynchrony we need a full stack requester going where there are multiple threads making IO requests. I think that's beyond the scope of this change so I think this test case can be deleted.

Reply:
@e-carlin Is the setup_module necessary for all cases? If so, that would be a separate issue/PR.

> Should we make an issue for the requester you envision, since I think the idea is that we want all our requests to eventually be asynchronous?

Yes, good idea. I started coding and then I deleted it. I think we need to have a lot of requests to the same simulation going that do different things, and ensure they all complete. I also want there to be some meta way about speaking about this similar to animation_test and report_test. I found the job_concurrency1_test too noisy.

I want to get the async tester going this month. This may address the issue naturally, because it will extensively test asynchrony.